src/api/app/controllers/webui/user_controller.rb

```diff
@@ -22,7 +22,8 @@ def show
     @role_titles = @displayed_user.roles.global.pluck(:title)
     @account_edit_link = CONFIG['proxy_auth_account_page']
 
-    return unless switch_to_webui2
+    return unless switch_to_webui2 && (CONFIG['contribution_graph'] == :on)
```

Comment:
Man I'm making a mess. this should be unless it's `:off` (so it's on by default). I'm so sorry...

Reply:
First of all, do we need this `:on` by default in `config/options.yml`? I have it `:off`.

On the other hand, I would say this line is ok :thinking: We should evaluate the lines after this condition (setting instance variables for the graph) only if we are in Bootstrap and we have the graph ON; otherwise it should return.

Reply:
It should be commented by default in options.yml. Like

```
# Disable the contribution graph on the user home page
# contribution_graph = :off 
```

Reply:
done!